src/containers/request.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @flow
+ */
+
+import xhr from 'xhr'
+
+export function isValidSlug(slug: any): boolean {
+  return !!slug || slug === 0
```

Comment:
why the `=== 0`?

Reply:
Because JS... Basically:

- We can't do just `!!slug` because the slug might be 0 (like an ID of 0)
- We can't do just `slug != null` because of `NaN`

Fortunately, NaN is also falsy, so by doing `!!slug` we work around NaN` and we do a direct comparison for 0.

Messy yeah?